src/shared/gpt.h

```diff
@@ -7,22 +7,38 @@
 
 #include "id128-util.h"
 
+#define GPT_ROOT_ALPHA       SD_ID128_MAKE(65,23,f8,ae,3e,b1,4e,2a,a0,5a,18,b6,95,ae,65,6f)
+#define GPT_ROOT_ARC         SD_ID128_MAKE(d2,7f,46,ed,29,19,4c,b8,bd,25,95,31,f3,c1,65,34)
```

Comment:
hmm, our ARCHITECTURE_xyz enum knows ARC_BE as well? And PPC_LE? and a couple of others? we might want to keep that list in sync

Reply:
I *can* add them, but I think I think they are not really used? If Debian doesn't have a triplet for them, then probably the hardware isn't available.